lavaplay/node_manager.py

```diff
@@ -171,11 +172,12 @@ async def search_soundcloud(self, query: str) -> t.Optional[t.Union[t.List[Track
             If the track could not be loaded.
         """
         result = await self.rest.load_tracks(f"scsearch:{query}")
-        if result["loadType"] == "NO_MATCHES":
+        res = result["data"]
+        if result["loadType"] == "empty":
```

Comment:
It not work

Reply:
I don't think it's right that it doesn't work the way it should.   ( https://lavalink.dev/api/rest.html#empty-result-data )

Reply:
I just tested it with a bot and it works.